src/components/banner/Banner.tsx

```diff
@@ -169,106 +127,50 @@ export const Banner = ({
+            <Button
+              className={styles.closeButton}
+              icon={
+                <RemoveIcon color={intentProps.removeIconColor} size={20} />
+              }
               onClick={onClose}
               variant="text"
             />
-          </Close>
+          </div>
         )}
 
-        <Header $color={removeIconColor}>{header}</Header>
+        <div
+          className={styles.header}
+          style={{ '--header-color': intentProps.removeIconColor }}
+        >
+          {header}
+        </div>
 
         {content && (
-          <Content>
+          <div className={styles.content}>
             {content}
             {moreContent}
-          </Content>
+          </div>
         )}
-      </Container>
+      </div>
     );
   };
 
-  switch (intent) {
-    case 'info':
-      return (
-        <InfoBanner className={className}>
-          {renderContent({
-            intentIcon: (
-              <InfoDuotoneIcon
-                color="blue800"
-                secondaryColor="blue400"
-                size={24}
-              />
-            ),
-            removeIconColor: 'blue800',
-          })}
-        </InfoBanner>
-      );
-    case 'success':
-      return (
-        <SuccessBanner className={className}>
-          {renderContent({
-            intentIcon: (
-              <CheckCircleDuotoneIcon
-                color="green800"
-                secondaryColor="green400"
-                size={24}
-              />
-            ),
-            removeIconColor: 'green800',
-          })}
-        </SuccessBanner>
-      );
-    case 'warning':
-      return (
-        <WarningBanner className={className}>
-          {renderContent({
-            intentIcon: (
-              <WarningDuotoneIcon
-                color="orange800"
-                secondaryColor="orange400"
-                size={24}
-              />
-            ),
-            removeIconColor: 'orange800',
-          })}
-        </WarningBanner>
-      );
-    case 'error':
-      return (
-        <ErrorBanner className={className}>
-          {renderContent({
-            intentIcon: (
-              <RemoveCircleDuotoneIcon
-                color="red800"
-                secondaryColor="red400"
-                size={24}
-              />
-            ),
-            removeIconColor: 'red800',
-          })}
-        </ErrorBanner>
-      );
-    case 'default':
-    default:
-      return (
-        <DefaultBanner className={className}>
-          {renderContent({
-            intentIcon: (
-              <InfoDuotoneIcon
-                color="gray800"
-                secondaryColor="gray400"
-                size={24}
-              />
-            ),
-            removeIconColor: 'gray800',
-          })}
-        </DefaultBanner>
-      );
-  }
+  const bannerClass = intent ? styles[`${intent}Banner`] : styles.defaultBanner;
+  return (
+    <div className={clsx([styles.styledBanner, bannerClass, className])}>
```

Comment:
Does it need to be array 🤔 
```suggestion
    <div className={clsx(styles.styledBanner, bannerClass, className)}>
```